website/src/components/SearchPage/SearchFullUI.tsx

```diff
@@ -174,6 +174,8 @@ export const InnerSearchFullUI = ({
     const aggregatedHook = hooks.useAggregated({}, {});
     const detailsHook = hooks.useDetails({}, {});
 
+    const [selectedSeqs, setSelectedSeqs] = useState<string[]>([]);
```

Comment:
Atm I'm often seeing that I tick one thing and the number increments by 2. We could use a `Set` which (A) probably makes sense (B) might fix this, depending on what the bug is.

Reply:
The bug should be fixed now, but a Set still makes sense I believe.

Reply:
https://github.com/loculus-project/loculus/pull/2917/commits/4bcc0b0e654be687aadb1bf3a6adc7c9bd884702